RELEASE/scripts/autoscend/quests/level_11.ash

```diff
@@ -2886,6 +2887,12 @@ boolean L11_palindome()
 	total = total + item_amount($item[Photograph Of God]);
 	total = total + item_amount($item[Photograph Of A Dog]);
 
+	if(isBanishedPhyla($phylum[dude]) && get_property("screechCombats").to_int() > 0)
```

Comment:
what is the purpose of checking `screechCombats`? Even if we can screech, there's no target in `autoscend_phylums.txt` to use it on and banish something besides dudes. Screech is complex though, maybe I'm missing something?

Reply:
When screech combats is greater than 0, that means that we have to do combat with the Eagle as our familiar. If it equals 0 that means that we can screech a new phylum. In the palindome, we target beasts to screech away to get rid of 3 of the 4 non dude monsters. Once beasts are screeched, the dudes will show back up. So if we can screech, we don't want to screech delay.